pkg/bloombuild/planner/metrics.go

```diff
@@ -86,6 +87,14 @@ func NewMetrics(
 			Help:      "Total number of tasks lost due to not being picked up by a builder and failed to be requeued.",
 		}),
 
+		planningTime: promauto.With(r).NewHistogram(prometheus.HistogramOpts{
+			Namespace: metricsNamespace,
+			Subsystem: metricsSubsystem,
+			Name:      "planning_time_seconds",
+			Help:      "Time spent during on planning during a build cycle.",
```

Comment:
Not quite sure what you wanted to say here:

```suggestion
			Help:      "Time spent planning a build cycle.",
```